Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -270,4 +253,29 @@ public void isReadyToPay(final Context context, final VenmoIsReadyToPayCallback
             }
         });
     }
+
+    private void callbackPaymentAuthAppSwitchFailure(VenmoPaymentAuthRequestCallback callback, VenmoPaymentAuthRequest request) {
+        braintreeClient.sendAnalyticsEvent(VenmoAnalytics.APP_SWITCH_FAILED);
+        callback.onVenmoPaymentAuthRequest(request);
+    }
+
+    private void callbackPaymentAuthFailure(VenmoPaymentAuthRequestCallback callback, VenmoPaymentAuthRequest request) {
+        braintreeClient.sendAnalyticsEvent(VenmoAnalytics.TOKENIZE_FAILED);
+        callback.onVenmoPaymentAuthRequest(request);
+    }
```

Comment:
If the app switch fails, I think we want that to count as a holistic "tokenize_failed", right? For that case on iOS we did [this](https://github.com/braintree/braintree_ios/blob/441f6d903ec8fd027459d124e61abb0b1b99f230/Sources/BraintreeVenmo/BTVenmoClient.swift#L346-L347), and send both a "conversion" event (tokenize_failed) + the "extra details" event (app_switch_failed).

This way we can use the "conversion" event of tokenize_failed to capture all failures, to use to measure against the total # of "start" events.

Though we went back and forth a few times on this with iOS, so definitely down to talk through this and come up with a pattern that makes sense across both.

Reply:
Ok agreed I think that makes sense - updated the app switch failure to send both the conversion analytic event and the app switch failure one. I agree with that approach for all the events 👍